react-rendering-lists/src/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import ReactDOM from "react-dom"
+
+
+
+const pokedex = [
+  { number: '001', name: 'Bulbasaur' },
+  { number: '004', name: 'Charmander' },
+  { number: '007', name: 'Squirtle' },
+  { number: '025', name: 'Pikachu' },
+  { number: '039', name: 'Jigglypuff' }
+];
+
+
+
+function PokemonList(props){
+  const pokemon = props.name;
+  const listItems = pokedex.map((pokemon) =>
```

Comment:
Line 17 will be `undefined` line 18 should be getting the array ok pokemon from props: 

```JSX
const listItems = props.pokemon.map
```

Make sure to console your variables and your props so you can see what is being stored in your variables